images.CI/macos/destroy-vm.ps1

```diff
@@ -0,0 +1,89 @@
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory)]
+    [ValidateNotNullOrEmpty()]
+    [string]$VMName,
+
+    [Parameter(Mandatory)]
+    [ValidateNotNullOrEmpty()]
+    [string]$VIServer,
+
+    [Parameter(Mandatory)]
+    [ValidateNotNullOrEmpty()]
+    [string]$VIUserName,
+
+    [Parameter(Mandatory)]
+    [ValidateNotNullOrEmpty()]
+    [string]$VIPassword
+)
+
+$ProgressPreference = "SilentlyContinue"
+$WarningPreference = "SilentlyContinue"
+
+# connection to a vCenter Server system
+try
+{
+    $null = Set-PowerCLIConfiguration -Scope Session -InvalidCertificateAction Ignore -ParticipateInCEIP $false -Confirm:$false -WebOperationTimeoutSeconds 600
+    $securePassword = ConvertTo-SecureString -String $VIPassword -AsPlainText -Force
+    $cred = New-Object System.Management.Automation.PSCredential($VIUserName, $securePassword)
+    $null = Connect-VIServer -Server $VIServer -Credential $cred -ErrorAction Stop
+    Write-Host "Connection to the VIServer has been established"
+}
+catch
+{
+    Write-Host "##vso[task.LogIssue type=error;]VISphere connection error"

```

Comment:
should it be `vSphere` instead of `VISphere`? and the same for other `VI`-like name
I am just a bit confused by naming

Reply:
Done. Thank you.